aiolifx/fixtures/base_fixture.py

```diff
@@ -0,0 +1,39 @@
+from dataclasses import dataclass
+
+from aiolifx.fixtures.device_features import DeviceFeatures
+
+
+# An interface ensuring every fixture has a capabilities list
+class RootFixture:
+    capabilities = [
+        DeviceFeatures.INFO,
+        DeviceFeatures.FIRMWARE,
+        DeviceFeatures.WIFI,
+        DeviceFeatures.UPTIME,
+        DeviceFeatures.REBOOT
+    ]
+    pass
```

Comment:
Do we need five features for this? I would probably roll them all up into `DeviceFeatures.BASE` because all LIFX devices respond to these requests and thus there's no real benefit in splitting them out into multiple entries.

Reply:
I use these same features as the menu options in the CLI so it can dynamically grab and then print options for the features/capabilities a device supports here:

https://github.com/aiolifx/aiolifx/pull/77/files#diff-e019bde2668955398e2ed9e23803b7a1c8ce14343f27b7000ea769d8a15ed8caR100-R129

Reply:
I think we should focus on bringing the library itself up to modern standards first, then consider the CLI. Particularly in terms of reducing the complexity that's developed over the past few years.

Realistically, the CLI should be refactored to become more scriptable to become more useful as a day-to-day tool.